src/renderers/common/Background.js

```diff
@@ -64,14 +64,14 @@ class Background extends DataMap {
 
 			// no background settings, use clear color configuration from the renderer
 
-			renderer._clearColor.getRGB( _clearColor, LinearSRGBColorSpace );
+			renderer._clearColor.getRGB( _clearColor );
```

Comment:
Better to not use `LinearSRGBColorSpace` and use the default `ColorManagement.workingColorSpace` which is `LinearSRGBColorSpace`.